src/components/Mastodon.vue

```diff
@@ -151,33 +159,33 @@
     }
     
     .article-image-wrapper {
-      width: calc(100% - 2rem);
-      height: 100%;
+
+//      flex: 1 0 calc(33% - #{$card-padding-internal * 2});
+
       display: block;
-      float: left;
-      background-color: $background;
-      background-size: cover; // this is probably temporary
-      background-position: center;
-      background-repeat: no-repeat;
-      margin: 0 $card-padding-internal 0 $card-padding-internal;
-      border-radius: $card_border-radius;
-      transition: all .5s ease-in-out;
-      flex: 1 0 calc(33% - #{$card-padding-internal * 2});
```

Comment:
Don't worry too much about Masto right now, that's my current task.

And since I'm sure this pops up elsewhere... see there. I moved them to a more global place, since these were defined very repetitively across templates.